src/common/channel/send-to-agnostic-channel-injection-token.ts

```diff
@@ -7,7 +7,7 @@ import type { Channel } from "./channel-injection-token";
 
 export type SendToAgnosticChannel = <TChannel extends Channel<unknown>>(
   channel: TChannel,
-  message: TChannel["_messageTemplate"]
+  message?: TChannel["_messageTemplate"]
```

Comment:
Why is message optional, that seems bad.

Reply:
There exists a use case for sending a signal with no message to trigger something (`broadcastThatRootFrameIsRenderedInjectable`).

Reply:
Yes, but a `void` parameter is optional. 

See [this playground link](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMOcAUAHAhgJywWwC5EA3OGAEwEpEBvAWACgmBfJpldKgbncZjBQApjmBYIQxAGEAFljBghAGwA8AWSEBnTVgDmQgHx0miRJWKaoOAbt6NTafFp37iG7XqFViZSnbbMjBAIlogAHsSy8oqqvhRGALyIAPIARgBWQtAAdFjaMLpgaGg0CUa0LAA0xvZmFMQARGENrFR8YSXcQA)

Reply:
Resolved for being obsolete. Note: a message payload is still optional, as not all messaging in Channels require payload; some are used as impulse/signal with no payload to simply trigger action.